lib/components/LinearProgressBar/__test__/LinearProgressBar.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render,screen } from '@testing-library/react';
+import { LinearProgressBar } from '..';
+
+describe('LinearProgressBar', () => {
+  it('renders with default props', () => {
+    render(
+      <LinearProgressBar
+        danger_limit={10}
+        is_loading={false}
+        label="Test Label"
+        percentage={50}
+        warning_limit={80}
+      />
+    );
+    const progressBar = screen.getByText("Test Label")
+    expect(progressBar).toBeInTheDocument();
+  });
+
+  it('renders loading state', () => {
+  render(
+      <LinearProgressBar
+        danger_limit={10}
+        is_loading={true}
+        label="Loading"
+        percentage={0}
+        warning_limit={80}
+      />
+    );
+    const loadingBar=document.querySelector(".deriv-progress-slider__infinite-loader");
+    expect(loadingBar).toBeInTheDocument();
+  });
+
+  it('renders progress bar with custom label', () => {
+    const { getByText } = render(
+      <LinearProgressBar
+        danger_limit={10}
+        is_loading={false}
+        label={<span>Custom Label</span>}
+        percentage={50}
+        warning_limit={80}
+      />
+    );
+    expect(getByText('Custom Label')).toBeInTheDocument();
```

Comment:
```suggestion
    const label = getByText('Custom Label'); 
    expect(label).toBeInTheDocument();
    expect(label.nodeName).to.Be('SPAN');
```
the purpose of this test case should be to check if the reactNode that we are passing as label is rendering correctly or not. so maybe we can also add this assertion as well. would you please check ? not sure if it has any errors or not.

Reply:
cc: @markwylde-deriv what do you think about this case? 🤔